ql/src/test/queries/clientpositive/load_static_ptn_into_bucketed_table.q

```diff
@@ -0,0 +1,33 @@
+set hive.stats.column.autogather=false;
+set hive.strict.checks.bucketing=true;
+
+set hive.explain.user=false;
+
+-- SORT_QUERY_RESULTS
+
+-- Single key partition
+-- Load with full partition spec
+CREATE TABLE src_bucket_tbl(key int, value string) partitioned by (ds string) clustered by (key) into 1 buckets STORED AS TEXTFILE;
+explain load data local inpath '../../data/files/bmj/000000_0' INTO TABLE src_bucket_tbl partition(ds='2008-04-08');
+load data local inpath '../../data/files/bmj/000000_0' INTO TABLE src_bucket_tbl partition(ds='2008-04-08');
+select * from src_bucket_tbl where ds='2008-04-08';
+
+drop table src_bucket_tbl;
+
+-- Multi key partition
+-- Load with both static and dynamic partition spec
+CREATE TABLE src_bucket_tbl(key int, value string) partitioned by (hr int, ds string) clustered by (key) into 1 buckets STORED AS TEXTFILE;
```

Comment:
Can we add another load test with multi-key static + dynamic partitioning, but with dynamic value taken from file instead of just null and ending in default partition? I believe ```load_data_1_partition.txt``` can be used for that purpose. Can you also invert partitioned keys (first static, second dynamic) in any of the tests?

Reply:
Updated test script as per your comment. Pls check.